services/management/ia/rules_service.go

```diff
@@ -92,8 +92,26 @@ type rule struct {
 	Annotations map[string]string   `yaml:"annotations,omitempty"`
 }
 
+// RemoveVMAlertRulesFiles removes all generated rules files (*.yml) on the ia path
+func (s *RulesService) RemoveVMAlertRulesFiles() error {
+	ruleFiles, err := filepath.Glob(rulesDir + "/*.yml")
+	if err == nil {
+
+		for _, file := range ruleFiles {
+			if err = os.RemoveAll(file); err != nil {
+				break
+			}
+		}
+
+		s.vmalert.RequestConfigurationUpdate()
+		s.alertManager.RequestConfigurationUpdate()
```

Comment:
We can remove it from here, it will be invoked anyway on line:
https://github.com/percona/pmm-managed/blob/5d330f5af2b646a0cf647315d236137edd3aca4d/services/server/server.go#L599